packages/shared/src/css/styles.css

```diff
@@ -1,6 +1,5 @@
 :root {
-  --p-transition-duration: 0s;
-  --p-animation-duration: 0s;
```

Comment:
Isn't this now breaking for consumers since it is public API in some components?